crates/polars-io/src/csv/buffer.rs

```diff
@@ -407,10 +416,16 @@ where
         _missing_is_null: bool,
         time_unit: Option<TimeUnit>,
     ) -> PolarsResult<()> {
-        if needs_escaping && bytes.len() > 2 {
```

Comment:
This check needs to remain, otherwise we can go out of bounds.

Reply:
already expected that.
I oriented myself at the other type implementations and they do not check either:
https://github.com/pola-rs/polars/blob/ed672fa2aa0c47dcc4fcd8a015e5f6c4bc514b1d/crates/polars-io/src/csv/buffer.rs#L84

If I put the check back in it **needs** to change to `>= 2` because otherwise an empty quoted string `""` would not become `null` but stay as type string

Reply:
@ritchie46 what do you think?

- keep as is (same as other types; no bounds check)
- change to `>=2` instead of `>2`

Reply:
> change to >=2 instead of >2

This one.